### PR TITLE
Zm unicode out

### PR DESCRIPTION
**Pull request by zacharym**
@thieman it might have been this simple

### CODE REVIEW

**Comment by zacharym** [comment]
@thieman I re-ran these puppies from earlier to show the tests work. Going to go through and add unicodes around the other fixtures

**Comment by thieman** [comment]
Cool

**Comment by zacharym** [comment]
@jsingle I think this takes care of us for encoding arguments, we still have inconsistency given how we are reading specs. cc @thieman

File: dusty/payload.py [hunk 1]
```diff
@@ -30,7 +38,9 @@ def serialize(self):
 
     @staticmethod
     def deserialize(doc):
-        return yaml.safe_load(doc)
+        loaded = yaml.safe_load(doc)
+        loaded['args'] = _encode_utf8(loaded['args'])
```
Comment (jsingle) [inline dusty/payload.py]: what about kwargs?

File: dusty/payload.py [hunk 2]
```diff
@@ -3,6 +3,14 @@
 
 from .constants import VERSION
 
+def _encode_utf8(args):
```
Comment (jsingle) [inline dusty/payload.py]: I'm concerned about having to traverse a data structure like this.  We should talk about it

Comment (zacharym) [inline dusty/payload.py]: I'll look into the root cause of the issues with `yaml.safe_load` and see if there is something we can do on that side

**Pull request closed without merging**